pkg/api/dtos/dashboard.go

```diff
@@ -7,34 +7,33 @@ import (
-	UpdatedBy                  string                `json:"updatedBy"`
-	CreatedBy                  string                `json:"createdBy"`
-	Version                    int                   `json:"version"`
-	HasACL                     bool                  `json:"hasAcl" xorm:"has_acl"`
-	IsFolder                   bool                  `json:"isFolder"`
-	FolderId                   int64                 `json:"folderId"`
-	FolderUid                  string                `json:"folderUid"`
-	FolderTitle                string                `json:"folderTitle"`
-	FolderUrl                  string                `json:"folderUrl"`
-	Provisioned                bool                  `json:"provisioned"`
-	ProvisionedExternalId      string                `json:"provisionedExternalId"`
-	AnnotationsPermissions     *AnnotationPermission `json:"annotationsPermissions"`
-	PublicDashboardAccessToken string                `json:"publicDashboardAccessToken"`
-	PublicDashboardUID         string                `json:"publicDashboardUid"`
-	PublicDashboardEnabled     bool                  `json:"publicDashboardEnabled"`
+	IsStarred              bool                  `json:"isStarred,omitempty"`
+	IsSnapshot             bool                  `json:"isSnapshot,omitempty"`
+	Type                   string                `json:"type,omitempty"`
+	CanSave                bool                  `json:"canSave"`
+	CanEdit                bool                  `json:"canEdit"`
+	CanAdmin               bool                  `json:"canAdmin"`
+	CanStar                bool                  `json:"canStar"`
+	CanDelete              bool                  `json:"canDelete"`
+	Slug                   string                `json:"slug"`
+	Url                    string                `json:"url"`
+	Expires                time.Time             `json:"expires"`
+	Created                time.Time             `json:"created"`
+	Updated                time.Time             `json:"updated"`
+	UpdatedBy              string                `json:"updatedBy"`
+	CreatedBy              string                `json:"createdBy"`
+	Version                int                   `json:"version"`
+	HasACL                 bool                  `json:"hasAcl" xorm:"has_acl"`
+	IsFolder               bool                  `json:"isFolder"`
+	FolderId               int64                 `json:"folderId"`
+	FolderUid              string                `json:"folderUid"`
+	FolderTitle            string                `json:"folderTitle"`
+	FolderUrl              string                `json:"folderUrl"`
+	Provisioned            bool                  `json:"provisioned"`
+	ProvisionedExternalId  string                `json:"provisionedExternalId"`
+	AnnotationsPermissions *AnnotationPermission `json:"annotationsPermissions"`
+	PublicDashboardUID     string                `json:"publicDashboardUid"`
+	PublicDashboardEnabled bool                  `json:"publicDashboardEnabled"`
```

Comment:
```suggestion
	PublicDashboardUID     string                `json:"publicDashboardUid,omitempty"`
	PublicDashboardEnabled bool                  `json:"publicDashboardEnabled,omitempty"`
```

just to keep things out of the payload when not in use